### PR TITLE
Removing stale windows packages

### PR DESCRIPTION
**Pull request by DominikDary**
## High-level description

Remove stale windows dcos packages to speed up build time and reduce complexity.

## Corresponding DC/OS tickets (required)

  - [DCOS-61811](https://jira.mesosphere.com/browse/DCOS-61811) Remove Stale Windows Packages.

## Checklist for component/package updates:

If you are changing components or packages in DC/OS (e.g. you are bumping the sha or ref of anything underneath `packages`), then in addition to the above please also include:

  - [ ] Change log from the last version integrated (this should be a link to commits for easy verification and review): [example](https://github.com/dcos/dcos-mesos-modules/compare/f6fa27d7c40f4207ba3bb2274e2cfe79b62a395a...6660b90fbbf69a15ef46d0184e36755881d6a5ae)
  - [ ] Included a test which will fail if code is reverted but test is not. If there is no test please explain.
  - [ ] Test Results: [link to CI job test results for component]
  - [ ] Code Coverage (if available): [link to code coverage report]

### CODE REVIEW

**Comment by DominikDary** [comment]
@mesosphere-mergebot sync

**Comment by DominikDary** [comment]
@mesosphere-mergebot changelog-not-required Cleanup for windows release

**Comment by DominikDary** [comment]
@mesosphere-mergebot request-review @tillt @jongiddy

File: packages/windows.treeinfo.json [hunk 1]
```diff
@@ -11,8 +11,6 @@
   },
   "exclude": [
     "erlang",
```
Comment (jongiddy) [inline packages/windows.treeinfo.json]: You should remove "erlang" here

Comment (jongiddy) [reply packages/windows.treeinfo.json]: Ah, sorry, I see that "erlang" still exists as a package.

**Pull request merged**